sklearn/ensemble/tests/test_bagging.py

```diff
@@ -73,9 +73,9 @@ def test_classification():
     ]
```

Comment:
In this file, we need to modify one estimator used for testing to follow our parameter validation:

```python
class DummySizeEstimator(BaseEstimator):
    def fit(self, X, y):
        self.training_size_ = X.shape[0]
        self.training_hash_ = joblib.hash(X)

    def predict(self, X):
        return np.ones(X.shape[0])
```

Reply:
I made this change to check that the rest of the tests are passing.